draft-ietf-httpbis-client-hints.md

```diff
@@ -89,7 +89,7 @@ Client Hints does not supersede or replace the User-Agent header field. Existing
 
 ## Notational Conventions
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in {{RFC2119}}.
+The key words "MUST", "MUST NOT", "REQUIRED", "SHOULD", "SHOULD NOT", "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14 {{RFC2119}} when, and only when, they appear capitalized, as shown.
```

Comment:
This doesn't seem to be consistent with https://tools.ietf.org/html/rfc8174#section-2